src/components/PersonLink.tsx

```diff
@@ -0,0 +1,62 @@
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const params = useParams();
+  const { sex, name, born, died, fatherName, motherName, slug } = person;
+
+  const parents = people.filter(
+    p => p.name === motherName || p.name === fatherName,
+  );
```

Comment:
Filtering the `people` array twice for `mother` and `father` is inefficient. Consider using a single pass to find both parents to improve performance.